commercetools-models/src/main/java/io/sphere/sdk/orders/OrderFromCartDraft.java

```diff
@@ -36,11 +52,32 @@
     @Nullable
     ShipmentState getShipmentState();
 
+    static OrderFromCartDraft of(final Cart cart, @Nullable final String orderNumber, @Nullable final PaymentState paymentState) {
+        ResourceIdentifier<Cart> cartResourceIdentifier = ResourceIdentifier.ofId(cart.getId());
+
+        return OrderFromCartDraftDsl.of(cartResourceIdentifier, cart.getVersion(), orderNumber, paymentState);
+    }
+
+    /**
+     * getId has been @deprecated so this method has been replaced with {@link}
+     */
+    @Deprecated
```

Comment:
This method can be removed as a Cart extends Versioned and instead refactor the other of method to create and use a resource identifier

Reply:
What is actually missing is a method `of` with a resource identifier as parameter.

Reply:
I didn't remove it because it could be possible that somebody has used it, isn't it?